openpilot/compile.py

```diff
@@ -7,8 +7,9 @@
 if os.getenv("GPU", None) is None:
```

Comment:
This would cause the test to fail, so I reverted back to os.getenv here.
```
if not getenv("OPT"):
  os.environ['OPT'] = '99'
if not getenv("GPU"):
  os.environ['GPU'] = '1'
```